src/components/Main.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BodyNavBar from './BodyNavBar';
+import Articles from './Articles';
+
+function Main() {
```

Comment:
Lets get rid of this component and move its content to the "Body" component. (which an earlier comment recommends renaming to Feed.)

Reply:
Done!